Transbank/Transbank.csproj

```diff
@@ -119,6 +119,10 @@
     <Folder Include="WebpayRest\WebpayPlus\" />
     <Folder Include="WebpayRest\WebpayPlus\Requests\" />
     <Folder Include="WebpayRest\WebpayPlus\Responses\" />
+    <Folder Include="WebpayRest\Oneclick\Request\" />
+    <Folder Include="WebpayRest\Oneclick\Response\" />
```

Comment:
These folders are in Singular. WebpayPlus ones are in Plural, and since there are several requests and responses inside we should keep them as Plural